### ISSUE TITLE
Conflict with Scrapbook X add-on

### ISSUE THREAD

**Issue by taman10** [gorhill/uBlock#913]
After updating from uBlock 1.3.3 to 1.3.4b2, my RMB context menu items for Scrapbook X disappeared. Disabled add-ons one-by-one to isolate conflict and found uBlock to be the one. Downgrading to 1.3.3 solves the problem. 

Firefox 42.0
Scrapbook X  - https://addons.mozilla.org/en-US/firefox/addon/scrapbook-x/

**Comment by gorhill** [comment]
You happen to have Tab Mix Plus?

If so, disable/enable "Make use of context menu where appropriate" in Settings.

**Comment by gorhill** [comment]
I pushed a new dev version 1.3.4b3 to the AMO store. Let me know if it helps.

This might be a duplicate of #906.

**Comment by taman10** [comment]
Thanks. 1.3.4b3 works!

**Comment by taman10** [comment]
Looks like I was premature in closing this issue. After restarting Firefox, the problem was back and the Scrapbook context menu items were missing. Only after unchecking "Make use of context menu where appropriate" in uBO and then restarting Firefox did the Scrapbook menu items came back.

Looks like 1.3.4b3 has the same problem as 1.3.4b2. Only way to avoid the menu items problem now is unchecking "Make use of context menu where appropriate".

**Comment by gorhill** [comment]
Try [latest dev build](https://addons.mozilla.org/en-US/firefox/addon/ublock-origin/versions/beta).

**Comment by gorhill** [comment]
Unable to reproduce using 1.3.4b5 (which has the fix).

**Comment by taman10** [comment]
Installed 1.3.4rc0, checked "Make use of context menu where appropriate" in uBO, closed and restarted Firefox several times. So far Scrapbook context menu items are displayed normally. The fix appears to be working!
